CarpetLib/src/restrict_3d_cc_rf2.cc

```diff
@@ -108,8 +108,8 @@ void restrict_3d_cc_rf2(T const *restrict const src,
 
   if (not use_loopcontrol_in_operators) {
 
-// Loop over coarse region
-#pragma omp parallel for collapse(3)
+    // Loop over coarse region
+    // #pragma omp parallel for collapse(3)
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.